it/repositories/AllowlistRepositorySpec.scala

```diff
@@ -96,4 +96,32 @@ class AllowlistRepositorySpec
       findAll().futureValue.size mustEqual 1
     }
   }
+
+  ".delete" - {
+
+    "must remove allow list entry when it exists within the database" in {
+
+      repository.set(entry1).futureValue
+      repository.set(entry2).futureValue
+      repository.delete(entry1).futureValue
+
+      findAll().futureValue must contain only entry2
+
+    }
+
+    "must not fail when the allow list entry doesn't exist within the database" in {
+
+      repository.delete(entry1).futureValue
+
+    }
+
+    "must successfully remove last allow list element" in {
```

Comment:
I think this test description could be clearer

Reply:
Sorted that. Let me know if that's OK! :)